### DISCUSSION TITLE
using / running composer. I'm confuse and lost, help me find my way :)

### DISCUSSION THREAD

**Discussion by Poeticsoul** [composer/composer#10359] [Q&A / Support]
I downloaded the installer, I ran it. when i type in composer and click on the program it runs something in CMD and closes. There is no program for me to use. My question is that normal if so how do I use composer. I have read info but it's a bit confusing so have basic instructions that's not top heavy will help.  If opening and closer of CMD is not normal what is the issue ?

Thank you.

**Comment by sovainfo** [comment]
Sounds like you are trying to execute it from a GUI. At least that is what clicking on it sounds to me. You should run it from a command shell. So, click the command shell and type in composer, providing one of the subcommands and parameters needed to execute that subcommand. Running without subcommand lists the available subcommands. Running it from a GUI causes that list of subcommands and returns to the GUI. That is expected, similar to clicking on a .bat, .cmd, .exe file.
